src/main/java/cart/domain/pay/PayPoint.java

```diff
@@ -0,0 +1,20 @@
+package cart.domain.pay;
+
+import cart.domain.vo.Money;
+
+import java.math.BigDecimal;
+import java.util.function.Function;
+
+public enum PayPoint {
```

Comment:
POLICY의 네이밍이 필요해보입니다-!

Reply:
```suggestion
public enum PointPolicy {
```

`PointPolicy`라는 네이밍으로 수정하여 포인트 정책임을 더 확실하게 나타냈습니다!